boxoffice/__init__.py

```diff
@@ -32,7 +35,7 @@ def init_for(env):
     coaster.app.init_app(app, env)
     db.init_app(app)
     db.app = app
-
+    baseframe.init_app(app, requires=['boxoffice'], ext_requires=['baseframe-bs3', 'fontawesome>=4.0.0'])
```

Comment:
This appears to remove dependencies that @vidya-ram has added. Please make sure you're not removing any.